includes/WooCommerceSupport.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace WCPress\WCP;
+
+// Security Check
+defined( 'ABSPATH' ) || exit;
+
+class WooCommerceSupport {
```

Comment:
Add doc block:
/**
 * Adds feature supports for WooCommerce 
 * 
 * @since WCP_SINCE
 */